catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchFilterVisitor.java

```diff
@@ -134,7 +135,8 @@ public Object visit(DWithin filter, Object data) {
             // we
             // can cast it
             PointImpl point = (PointImpl) literalWrapper.evaluate(null);
-            double[] coords = point.getCentroid().getCoordinate();
+            double[] coords = point.getCentroid()
+                    .getCoordinate();
             double distance = filter.getDistance();
 
             logger.debug("point: coords[0] = " + coords[0] + ",   coords[1] = " + coords[1]);
```

Comment:
Logger is using string concatenation instead of {} substitution. Perhaps it should be changed for consistency?